api/swagger.yaml

```diff
@@ -1362,6 +1867,14 @@ parameters:
     description: |
       Comma-separated list of event types. The resulting list will only have events of the given types, events of other types will be ommitted.
     required: false
+  groupingIdParam:
+    name: groupingId
+    in: query
+    type: string
+    format: comma-separated
+    description: |
+      Comman-separated list of `groupingId`s. The resulting list will only have events with the given grouping IDs.
```

Comment:
Typo: Comman => Comma

Reply:
What is a typical use case for `groupingId`?

Reply:
Nvm, it's explained below.

Reply:
In query parameters, document that more than one groupingId is allowed and each id will be orred on the filter.